operator/v1/types.go

```diff
@@ -59,6 +63,23 @@ type OperatorSpec struct {
 	ObservedConfig runtime.RawExtension `json:"observedConfig"`
 }
 
+type LogLevel string
+
+var (
+	// Normal is the default.  Normal, working log information, everything is fine, but helpful notices for auditing or common operations.
+	Normal LogLevel = "Normal"
```

Comment:
I'm also missing higher level Error/Warning type of.

Reply:
> I'm also missing higher level Error/Warning type of.

I don't see a reason to encourage that, but if we were to do it, `Minimal` seems more likely.

Reply:
> but if we were to do it, Minimal seems more likely.

Hmm :( `Minimal/Normal/Trace/TraceAtAllCost` are more constants that are specific to us.. is there a benefit for users by re-defining these things... ? 

An example of already of what is accepted is:
From https://tools.ietf.org/html/rfc3164#section-4.1.1
```
        Numerical         Severity
          Code

           0       Emergency: system is unusable
           1       Alert: action must be taken immediately
           2       Critical: critical conditions
           3       Error: error conditions
           4       Warning: warning conditions
           5       Notice: normal but significant condition
           6       Informational: informational messages
           7       Debug: debug-level messages
```

Reply:
> I don't see a reason to encourage that, but if we were to do it, `Minimal` seems more likely.

SGTM

Reply:
I also agree with David here, we don't have to blindly follow the standard, since these are very dependent on our implementation, but I agree we should come close enough to be easily distinguishable. Each library has its own levels, let's try to be somewhat consistent.